src/bundle/Resources/public/js/scripts/admin.location.view.js

```diff
@@ -7,6 +7,7 @@
     const sortContainer = doc.querySelector('[data-sort-field][data-sort-order]');
     const sortField = sortContainer.getAttribute('data-sort-field');
     const sortOrder = sortContainer.getAttribute('data-sort-order');
+    const contentName = doc.querySelector('.ez-sil').dataset.contentName;
```

Comment:
Don't think this should be in the subitems div. This should be stored in the modal itself to be independent.